### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by Omgise**

### CODE REVIEW

File: src/main/resources/assets/aether_legacy/lang/zh_CN.lang
Comment (AetherLegacyPro) [inline src/main/resources/assets/aether_legacy/lang/zh_CN.lang:1]: Thanks! I will add this to the patch later today.

**Comment by Richard-Rogalski** [comment]
Thanks Omgise for the extensive contribution!!!!

Also @AetherLegacyPro , you could just hit the 'merge pull request' button lmfao

**Comment by KELETU66666** [comment]
可以的老哥，方便加个qq吗，你这个太牛皮了

**Comment by Omgise** [comment]
> 可以的老哥，方便加个qq吗，你这个太牛皮了

在MC百科私信你了

**Pull request closed without merging**